tests/unit/mocks.py

```diff
@@ -236,6 +238,55 @@ async def _get_client_certificate(
         ).decode("UTF-8")
         return (server_cert, [client_cert, intermediate_cert, root_cert])
 
+    async def get_connection_info(
```

Comment:
will have a follow-up PR to clean up the `FakeAlloyDBClient` mock to remove this copy/paste code.